core/templates/pages/contributor-dashboard-page/contributions-and-review/contributions-and-review.component.spec.ts

```diff
@@ -1420,8 +1421,11 @@ describe('Contributions and review component', () => {
           subheading: 'topic_name / story_title / chapter_title',
           labelText: 'Awaiting review',
           labelColor: '#eeeeee',
-          actionButtonTitle: 'Review'
+          actionButtonTitle: 'Review',
+          translationLengthInWords: undefined
```

Comment:
I would expect this to not be `undefined`. I think we need to update this test to have translations data.